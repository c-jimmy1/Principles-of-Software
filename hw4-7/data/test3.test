r
EMPAC
Academy Hall
r
Lally Hall
EMPAC
q
