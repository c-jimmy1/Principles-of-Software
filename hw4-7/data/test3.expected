First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from EMPAC to Academy Hall:
	Walk East to (MRC)
	Walk East to (Empire State Hall)
	Walk East to (Cogswell Laboratory)
	Walk East to (Intersection 137)
	Walk SouthEast to (Academy Hall)
Total distance: 490.489 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from Lally Hall to EMPAC:
	Walk South to (Folsom Library)
	Walk SouthWest to (EMPAC)
Total distance: 233.812 pixel units.
